com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/searcher/TestFrameworkSearcher.java

```diff
@@ -13,14 +13,17 @@
 
 import com.microsoft.java.test.plugin.model.TestKind;
 
+import org.eclipse.jdt.core.IMethod;
 import org.eclipse.jdt.core.search.SearchPattern;
 
-public abstract class JUnitTestSearcher {
+public interface TestFrameworkSearcher {
```

Comment:
This is a good start. But I don't see getSearchPattern and getTestMethodAnnotation called anywhere. Do we really need those two?

Reply:
Removed those unused methods.